### ISSUE TITLE
Audio compressor

### ISSUE THREAD

**Issue by rikai** [JupiterBroadcasting/CasterSoundboard#53]
Since some clips come in with different volumes, it'd be nice if we could apply some kind of compressor effect (toggleable) to bring low clips up and high clips down.

**Comment by odysseywestra** [comment]
Hairbrain Idea. Would having hooks to PulseEffects work in this case? Doesn't have to be a dependency, but using it like a plugin would benefit. At least then we wouldn't have to build a whole framework and have the equalizer stuff we would need.

**Comment by rikai** [comment]
I was actually looking at something neat like [this](https://github.com/lordmulder/DynamicAudioNormalizer) ...it seems pretty damn awesome, and capable of realtime stream processing from what i've read so far...

**Comment by odysseywestra** [comment]
Yeah, having an built in option for dynamic normalization would benefit soundboard. Probably want to change your title then to `Add Support for Dynamic Audio Normalizer` to narrow down the Feature Request.

**Comment by rikai** [comment]
Perhaps, but not yet. There may be other, better options.
